### PR TITLE
7 new matches, and code cleanup

### PR DESCRIPTION
**Pull request by Ryan-Myers**
Matched several more funcs, and did some more code cleanup.

### CODE REVIEW

File: src/textures_sprites.c [hunk 1]
```diff
@@ -1233,7 +1233,78 @@ s32 func_8007EF64(s16 arg0) {
...
+    s32 bit26Set;
+    s32 phi_a0;
+
+    bit23Set = *triangleBatchInfoFlags & 0x800000;
+    bit26Set = *triangleBatchInfoFlags & 0x04000000;
+    bit25Set = *triangleBatchInfoFlags & 0x02000000;
+    if (bit23Set) {
+        if (!bit25Set) {
+            if (get_random_number_from_range(0, 1000) >= 986) {
+                *triangleBatchInfoFlags &= 0xFBFFFFFF; //Unset bit 26
```
Comment (Faschz) [inline src/textures_sprites.c:1248]:
```suggestion
                *triangleBatchInfoFlags &= ~0x4000000;
```

File: src/textures_sprites.c [hunk 2]
```diff
@@ -1233,7 +1233,78 @@ s32 func_8007EF64(s16 arg0) {
...
+                *triangleBatchInfoFlags |= 0x02000000; //Set bit 25
+            }
+        } else if (!bit26Set) {
+            *arg2 = *arg2 + (texture->frameAdvanceDelay * arg3);
+            if (*arg2 >= texture->numOfTextures) {
+                *arg2 = ((texture->numOfTextures * 2) - *arg2) - 1;
+                if (*arg2 < 0) {
+                    *arg2 = 0;
+                    *triangleBatchInfoFlags &= 0xF9FFFFFF; //Unset bits 25 and 26
```
Comment (Faschz) [inline src/textures_sprites.c:1257]:
```suggestion
                    *triangleBatchInfoFlags &= ~(0x2000000 | 0x4000000);
```

File: src/textures_sprites.c [hunk 3]
```diff
@@ -1233,7 +1233,78 @@ s32 func_8007EF64(s16 arg0) {
...
+                    return;
+                }
+                *triangleBatchInfoFlags |= 0x04000000; //Set bit 26
+            }
+        } else {
+            *arg2 = *arg2 - (texture->frameAdvanceDelay * arg3);
+            if (*arg2 < 0) {
+                *arg2 = 0;
+                *triangleBatchInfoFlags &= 0xF9FFFFFF; //Unset bits 25 and 26
```
Comment (Faschz) [inline src/textures_sprites.c:1266]:
```suggestion
                *triangleBatchInfoFlags &= ~(0x2000000 | 0x4000000);
```

File: src/textures_sprites.c [hunk 4]
```diff
@@ -1233,7 +1233,78 @@ s32 func_8007EF64(s16 arg0) {
...
+            if (!bit26Set) {
+                *arg2 += texture->frameAdvanceDelay * arg3;
+            } else {
+                *arg2 -= texture->frameAdvanceDelay * arg3;
+            }
+            do {
+                phi_a0 = 0;
+                if (*arg2 < 0) {
+                    *arg2 = -*arg2;
+                    *triangleBatchInfoFlags &= 0xFBFFFFFF; //All but bit 26
```
Comment (Faschz) [inline src/textures_sprites.c:1280]:
```suggestion
                    *triangleBatchInfoFlags &= ~0x4000000;
```

File: src/camera.c [hunk 5]
```diff
@@ -275,20 +275,20 @@ void func_80066230(Gfx **dlist, Gfx **arg1) {
...
     }
 
-    temp_f0 = arg2 - D_80120AC0[phi_v1].trans.z_position;
-    temp_f2 = arg0 - D_80120AC0[phi_v1].trans.x_position;
-    temp_f14 = arg1 - D_80120AC0[phi_v1].trans.y_position;
-    return sqrtf((temp_f0 * temp_f0) + ((temp_f2 * temp_f2) + (temp_f14 * temp_f14)));
+    z = zPos - D_80120AC0[index].trans.z_position;
+    x = xPos - D_80120AC0[index].trans.x_position;
+    y = yPos - D_80120AC0[index].trans.y_position;
+    return sqrtf((z * z) + ((x * x) + (y * y)));
```
Comment (Faschz) [inline src/camera.c:291]: Perhaps these should be called dx, dy, and dz since they are differences in position.

**Review by AntonioCastelli — APPROVED** [review]

**Pull request merged**